### DISCUSSION TITLE
Page history not working in 0.6.2

### DISCUSSION THREAD

**Discussion by elsheep** [docmost/docmost#567] [Help / Questions] [answered]
Edited some page and logout.

After about half hour, login the system, all pages still have nothing in "page history"

![图片](https://github.com/user-attachments/assets/5b570930-9a65-4dd5-bc6e-7f18761b9da3)

**Comment by elsheep** [comment]
BTW, version I used is 0.6.2, and comment function also not working

**Reply by Philipinho** [reply]
Are you still having issues with commenting? it is inline commenting, you have to highlight the text to comment on.

**Reply by elsheep** [reply]
Found and verified. Thanks.

**Comment by Philipinho** [comment]
Please check the websocket "WS" tab on your browser and confirm if the `/collab `path transmit data when you type on the editor.

**Comment by elsheep** [comment]
WS is working when editing the page.
![图片](https://github.com/user-attachments/assets/0b953d31-994e-49ea-b7ac-049943dc5056)

And found a wiered act that, after about an hour, history page appeared in 2 of 4 pages that I edited.

Don't know why the rest 2 pages have no history

**Reply by Philipinho** [reply, marked as answer]
The page history is saved after every 10 minutes of active edits. Sometimes, the last edits is only saved the next time you edit the document.

**Reply by elsheep** [reply]
Thank you for your answer. I also found out that sometimes I open the page and edit, there's no ws alive.
